src/components/countyDashboard.tsx

```diff
@@ -31,6 +34,8 @@ const DashboardControls: React.FC<ControlsPropsType> = (props) => {
 const CountyDashboard: React.FC = () => {
   const [currState, setCurrState] = useState(stateData[0]);
   const [currCounty, setCurrCounty] = useState(countyData[stateData[0] as CountyName][0]);
+  const [currData, setCurrData] = useState(dataTypes[0] as DataType[0]);
+  const [currLink, setCurrLink] = useState(dataLinks[dataTypes[0] as DataType]);
```

Comment:
the `as` keyword forced the value to be of the following type which could cause unpredictable results; therefore, it is advisable to avoid using `as`.
```suggestion
  const [currCounty, setCurrCounty] = useState<string>(countyData[stateData[0]][0]);
  const [currData, setCurrData] = useState<DataType>(dataTypes[0]);
  const [currLink, setCurrLink] = useState<string[]>(dataLinks[dataTypes[0]]);
```